client/src/app/users/user-list.component.ts

```diff
@@ -32,6 +32,7 @@ export class UserListComponent implements OnInit, OnDestroy  {
   public userCompany: string;
   public viewType: 'card' | 'list' = 'card';
 
+  errMsg = '';
```

Comment:
Is there a reason you moved this out of `getUsersFromServer()`? It's only used there, so it seems that it would make more sense as a local variable there. Is it necessary to ensure that the message passes through to the snackbar in some way?

Oh, I think I get it now. You needed it to be at the class level so you could access it in the tests. Is that right? If so, we might want to document that.